velox/functions/lib/string/StringImpl.h

```diff
@@ -640,4 +640,81 @@ FOLLY_ALWAYS_INLINE void pad(
       padPrefixByteLength);
 }
 
+FOLLY_ALWAYS_INLINE static void
+initCapAscii(char* output, const char* input, size_t length) {
+  bool newWord = true;
+  for (size_t i = 0; i < length; ++i) {
+    if (std::isspace(input[i])) {
+      output[i] = input[i];
+      newWord = true;
+    } else {
+      output[i] = newWord ? std::toupper(input[i]) : std::tolower(input[i]);
+      newWord = false;
+    }
+  }
+}
+
+FOLLY_ALWAYS_INLINE size_t initCapUnicode(
```

Comment:
@xanderbailey, can we just reuse the below code? https://github.com/facebookincubator/velox/blob/main/velox/functions/lib/string/StringCore.h#L120, which handles all input, but maybe we can pass the initial character for each word?

Reply:
Just noted remaining characters in a word needs to be converted to lower case. So we cannot directly re-use it.